docs/zh_cn/migration.md

```diff
@@ -0,0 +1,169 @@
+#  MMTracking 版本迁移
+
+与 0.xx  版本的 MMTracking 相比, 最新的 1.xx 版本有以下重要修改。

```

Comment:
Please note that the one blank space between **English & numbers** and **Chinese**. Check all the docs.